examples/complete/main.tf

```diff
@@ -75,6 +75,13 @@ module "eventbridge" {
       {
         name = "log-orders-to-cloudwatch"
         arn  = aws_cloudwatch_log_group.this.arn
+      },
+      {
```

Comment:
This won't fit here since there is a limit of 5 targets per rule, I will fix it now and merge.